packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -89,6 +89,11 @@ typedef AutocompleteOptionToString<T extends Object> = String Function(T option)
 
 /// A direction in which to open the options-view overlay.
 ///
+/// The available space for the options view is calculated based on the
+/// constraints of the [Overlay] that contains the autocomplete widget,
```

Comment:
nit: constaints -> size

Reply:
Also this is only relevant to `automatic` so consider putting the docs there instead